packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.flow.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import type { Globals } from "../../../common/common.js.flow";
+
+export type Props = {|
+  shown: boolean,
+  dialogId: ?string,
+  children: React$Node,
+  parent: ?React$Node,
+  onClose: () => void,
+  ...Globals,
+|};
+
+declare export default React.ComponentType<Props>;
```

Comment:
Should there be a TypeScript definition for this? Probably not if it's a private component, right?

Reply:
Exactly as it is not intended to be exported then flow should be sufficient.